.circleci/config.yml

```
@@ -63,45 +66,33 @@ jobs:
           command: |
             cd test/integration
             dbt compile --target sqlserver_local_encrypt
-      - run:
-          name: cnxn -- Azure SQL - SQL cred
-          command: |
-            cd test/integration
-            dbt compile --target azuresql_sqlcred
 
-      # TODO drop support for this method in favor of
-      # https://github.com/dbt-msft/dbt-sqlserver/pull/71
-      # - run:
-      #     name: cnxn -- Azure SQL - AAD Password
-      #     command: |
-      #       cd test/integration
-      #       dbt compile --target azuresql_aad
   integration-azuresql:
     docker:
       - image: dataders/pyodbc:1.4
     executor: python/default
```

last thought is this line still needed? Not sure what it does anymore. esp because we're running bash commands on a python-base docker image
```suggestion
    executor: python/default
```